build.gradle

```diff
@@ -200,8 +200,7 @@ publishing {
 
 repositories {
     mavenLocal()
-    maven { url "https://central.sonatype.com/repository/maven-snapshots/" }
```

Comment:
Lets keep the "https://central.sonatype.com/repository/maven-snapshots/" under repositories to retrieve older artifacts. This way things wont break.

Reply:
We dont need to do that anymore.
I already moved 42G of old artifacts to the new endpoint.
Adding that will just add up to confusion, if we have something missing just backport the PR to that branch and re-publish.